include/asterisk/_private.h

```diff
@@ -57,6 +57,7 @@ int ast_msg_init(void);             /*!< Provided by message.c */
 void ast_msg_shutdown(void);        /*!< Provided by message.c */
 int aco_init(void);             /*!< Provided by config_options.c */
 int dns_core_init(void);        /*!< Provided by dns_core.c */
+int ast_refer_init(void);             /*!< Provided by refer.c */
```

Comment:
refer isn't something that really has existed in the core, we commonly use transfer to refer to this - is there a particular reason you chose refer instead of that, or was it just because this was driven by SIP?

Reply:
We wanted to capture the idea of referring an endpoint to some resource, without requiring an ongoing "session" with that endpoint. We thought transfer could be confusing as it might imply that there is already a "session". But I'm open to renaming it.
And yes, it was driven by SIP.

Reply:
I don't have a better word for it, so we can stick with refer.